api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -169,6 +174,20 @@ public void saveNotebook(String bucketName, String notebookName, JSONObject note
         notebookContents.toString().getBytes(StandardCharsets.UTF_8));
   }
 
+  @Override
+  public String getReadOnlyHtml(
+      String workspaceNamespace, String workspaceName, String notebookName) {
+    String bucketName =
```

Comment:
nit: probably worth commenting that this call here is what is enforcing the ACLs for the notebook bucket

Reply:
Where is this check? Also I do think calling `getWorkspace()` is sufficient - we should avoid making two serial calls to FC if we can avoid it.

Reply:
Nevermind, I must have been looking at a stale commit, I see it now. Second statement still stands though